src/test/java/tracko/testutil/TypicalItems.java

```diff
@@ -13,19 +14,33 @@
 public class TypicalItems {
 
     public static final Item ITEM_1 = new ItemBuilder().withItemName("Sofa")
-            .withQuantity(200).withDescription("Made of leather").build();
+            .withQuantity(200).withDescription("Made of leather")
```

Comment:
I think you can maybe consider putting sell/cost price of typical item shere with different amount of decimal places to introduce a variety of data.

For example, instead of 50, we can maybe use 49.99, etc, so that we can be sure that the application can appropriately handle cost/sell prices with various decimal points!